packages/tatum-core/src/model/response/offchain/Wallet.ts

```diff
@@ -0,0 +1,25 @@
+export type Wallet = WalletWithMnemonic | WalletWithAddress
+
+export interface WalletWithMnemonic {
```

Comment:
Should we also move this interface to define library? There is a problem that tatum-ledger also needs these interfaces but we can add dependency to tatum-ledger on tatum-defi.

Reply:
move to ledger pls and add as dependency to defi

Reply:
@prokopsimek I would add tatum-core as a dep for tatum-ledger, thats ok?

Reply:
@Hathoriel `tatum-core` already is a dependency of `tatum-ledger`.